src/components/views/messages/IBodyProps.ts

```diff
@@ -55,4 +55,8 @@ export interface IBodyProps {
     getRelationsForEvent?: GetRelationsForEvent;
 
     ref?: React.RefObject<any> | LegacyRef<any>;
+
+    // Set to `true` to disable interactions (e.g. video controls) and tab indexes.
```

Comment:
I'm still not really sure what a tab index is. That might just be me though?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

We tend to expect a base knowledge of web primitives in our comments normally

Reply:
Fair, and this is probably just my lack of web dev experience coming through, but even having read the MDN page, I'm not entirely sure what "disabling a tab index" means. Does it mean making it untabbable (but not necessarily unfocussable) - ie, the effect of setting `tabindex=-1` ? Why is that better than just disabling the controls?

In any case, we don't actually seem to do that in this PR?

Reply:
> ie, the effect of setting tabindex=-1 ?

Yes

> Why is that better than just disabling the controls?

For video disabling the controls is good, but for buttons and other interactive elements you can't specify `controls=false` - you use `disabled=true` which to accessible technologies makes it still take up an index in the tab list which can be confusing depending on the context, like when we're (ab)using the EventTile in a non-interactive element such as the preview in the appearance tab

> In any case, we don't actually seem to do that in this PR?

Indeed, it was already done, the prop isn't new, just being passed more widely.

Reply:
Ok, I can't find anywhere we currently set the tabindex to -1, but I'll take your word for it that it happens and is a thing we want to do.

I've read https://developer.mozilla.org/en-US/docs/Web/Accessibility/Keyboard-navigable_JavaScript_widgets, and I can't find anywhere this is referred to as "disabling a tab index". Rather, it talks about "removing a control from the tab order" or making an element "not tab navigable". (Indeed, adding a `tabindex=-1` attribute to an element sounds like we are *adding* a tab index, not disbling one -- even if that tab index is negative).

So, I heavily suspect this usage of "tab index" is something that has been dreamt up within the team; we should avoid such concepts where possible to make it easy for new members (hi!) to onboard.

So, YMMV, but IMHO this is clearer:

```suggestion
    // Set to `true` to disable interactions (e.g. video controls) and to remove controls from the tab order.
```

Reply:
> Ok, I can't find anywhere we currently set the tabindex to -1, but I'll take your word for it that it happens and is a thing we want to do.

<img width="975" alt="image" src="https://github.com/matrix-org/matrix-react-sdk/assets/2403652/53217b46-de45-4b50-8ddb-50f3443dd010">

Reply:
> <img alt="image" width="975" src="https://user-images.githubusercontent.com/2403652/243611633-53217b46-de45-4b50-8ddb-50f3443dd010.png">

ok, but are any of those anything to do with `inhibitInteraction`?